src/StaticFAQ/FAQArticleDetailContent.js

```diff
@@ -5,58 +5,65 @@ import Markdown from 'markdown-to-jsx';
 import { Typography } from '@kiwicom/orbit-components';
 import { createFragmentContainer, graphql } from 'react-relay';
 
-import type { FAQArticleDetailContent_article } from './__generated__/FAQArticleDetailContent_article.graphql';
+import Breadcrumbs from './Breadcrumbs';
+import type { FAQArticleDetailQuery } from './__generated__/FAQArticleDetailQuery.graphql';
 
-type Props = {
-  article: FAQArticleDetailContent_article,
-};
+type Props = FAQArticleDetailQuery;
```

Comment:
this looks suspicious, is there `FAQArticleDetailQuery` type?

old `article: FAQArticleDetailContent_article` should be kept, that's why it was generated